daemon/health.go

```diff
@@ -81,6 +81,8 @@ func (p *cmdProbe) run(ctx context.Context, d *Daemon, cntr *container.Container
 	execConfig.Privileged = false
 	execConfig.User = cntr.Config.User
 	execConfig.WorkingDir = cntr.Config.WorkingDir
+	// health checks don't get the full 10 seconds timeout for a .
```

Comment:
whoops; looks like I was still writing here but didn't finish that before opening https://github.com/ndeloof/docker/pull/2

Trying to recall what I wanted to write here; I guess something about stopping gracefully.

```go
// health checks don't get the full 10 seconds timeout to stop gracefully.
```